virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,71 +1,44 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Attaching a virtual machine to a Linux bridge network
 include::_attributes/common-attributes.adoc[]
-:context: virt-attaching-multiple-networks
+:context: virt-attaching-vm-multiple-networks
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+By default, {VirtProductName} is installed with a single, internal pod network.
 
-To attach a virtual machine to an additional network:
-
-. Configure a Linux bridge.
-. Configure a bridge network attachment definition for a namespace in the web console or CLI.
-+
-[NOTE]
-====
-The network attachment definition must be in the same namespace as the pod or virtual machine.
-====
-. Attach the virtual machine to the network attachment definition by using either the web console or the CLI:
-** In the web console, create a NIC for a new or existing virtual machine.
-** In the CLI, include the network information in the virtual machine configuration.
-
-[NOTE]
-====
-There are multiple methods for configuring a VLAN, including network attachment definition and node network configuration policy.
-However, a network attachment definition provides a more efficient and more manageable configuration.
-====
+You must create a Linux bridge network attachment definition (NAD) in order to connect to additional networks.
 
-include::modules/virt-networking-glossary.adoc[leveloffset=+1]
+= Connecting to the network through the network attachment definition
 
-[id="virt-creating-linux-bridge"]
-== Configuring a Linux bridge
+To attach a virtual machine to an additional network:
 
-include::modules/virt-creating-linux-bridge-nncp.adoc[leveloffset=+2]
```

Comment:
Was this an intentional deletion? You have changes to the `virt-creating-linux-bridge-nncp.adoc` file in this PR, but I couldn't find it included anywhere to preview the changes. I didn't see it included in another assembly (that I could find at least)

Reply:
I think that's correct @bergerhoffer .... it was deleted. I also moved some of the introductory text up to the top of the assembly.... it was under the first head2 and that didn't make much sense. I changes a lot the levels to H3s where needed. I also put the field table at the bottom. 

@apinnick - Let me get our confirmation on these changes from the preview build. Thanks.

Reply:
Bob, I think you can leave the networking field table, since it is relevant for the NIC/UI section. I would change the "Networking fields" header to "discrete". I leave that decision to you.

Reply:
I re-added the networking fields module, but didn't change anything else.